lib/manager/nuget/extract.ts

```diff
@@ -1,18 +1,22 @@
 import { logger } from '../../logger';
 import { get } from '../../versioning';
-import { PackageDependency, ExtractConfig, PackageFile } from '../common';
+import {
+  PackageDependency,
+  PackageFile,
+  ExtractPackageFileConfig,
+} from '../common';
 
-export function extractPackageFile(
-  content: string,
-  packageFile: string,
-  config: ExtractConfig = {}
-): PackageFile {
-  logger.trace(`nuget.extractPackageFile(${packageFile})`);
+export function extractPackageFile({
+  fileContent,
+  fileName,
+  config = {},
+}: ExtractPackageFileConfig): PackageFile {
+  logger.trace(`nuget.extractPackageFile(${fileName})`);
   const { isVersion } = get(config.versionScheme || 'semver');
```

Comment:
```suggestion
  const { isVersion } = get(config?.versionScheme || 'semver');
```

Reply:
prettier treats this as a syntax error.  :expressionless: